src/osd/ClassHandler.h

```diff
@@ -34,11 +34,11 @@ class ClassHandler
       return flags;
     }
 
-    ClassMethod() : cls(0), flags(0), func(0), cxx_func(0) {}
+    ClassMethod() : flags(0), func(0), cxx_func(0) {}
```

Comment:
Maybe `... = default` as we're already touching the line?

Reply:
@rzarzynski i updated this class to use `std::variant`, could you please take another look?